### DISCUSSION TITLE
Digital AI microphone

### DISCUSSION THREAD

**Discussion by Vertikar** [tphakala/birdnet-go#261] [General]
Just saw this pop up in my feed, could be interesting to use. Didn't see the frequency range mentioned though.

https://www.cnx-software.com/2024/08/20/anavi-dev-mic-digital-omnidirectional-microphone-raspberry-pi-rp2040-mcu/?amp=1

**Comment by tphakala** [comment]
specs say "Sample Rate – 16 kHz", which means frequency response up to 8 kHz, which is pretty bad for BirdNET use.
